src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -881,9 +892,9 @@ private bool IsFailOverable(AggregateException ex)
         {
             IReadOnlyCollection<Exception> innerExceptions = ex.InnerExceptions;
```

Comment:
Can you also change `ExecuteWithFailOverPolicyAsync` method to catch AggregateException after other exceptions?